uReplicator-Controller/src/main/java/com/uber/stream/kafka/mirrormaker/controller/ControllerInstance.java

```diff
@@ -228,18 +228,6 @@ public boolean isStarted() {
    */
   public boolean stop() {
     boolean success = true;
-    started = false;
```

Comment:
what's the reason to delete this session. 
I believe this is useful for clean shutdown he controllerInstance

Reply:
i move the close logic of KafkaBrokerTopicObserver in *ValidationManager.stop。
so we don't need to close KafkaBrokerTopicObserver in the container。